src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -173,88 +198,116 @@ subroutine readParams ( ncid )
     type(file_desc_t),intent(inout) :: ncid   ! pio netCDF file id
     !
     ! !LOCAL VARIABLES:
-    character(len=32)  :: subname = 'CNDecompBgcParamsType'
-    character(len=100) :: errCode = 'Error reading in CN const file '
+    character(len=32)  :: subname = 'readParams'
+    character(len=100) :: errCode = 'Error reading MIMICS params '
     logical            :: readv   ! has variable been read in or not
     real(r8)           :: tempr   ! temporary to read in constant
     character(len=100) :: tString ! temp. var for reading
     !-----------------------------------------------------------------------
 
     ! Read off of netcdf file
-    tString='tau_l1'
+    ! TODO slevis: new params will be added here and in the params file
+    ! TODO slevis: Keep tau_cwd?
+    tString='tau_cwd'
     call ncd_io(trim(tString),tempr, 'read', ncid, readvar=readv)
     if ( .not. readv ) call endrun(msg=trim(errCode)//trim(tString)//errMsg(sourcefile, __LINE__))
-    params_inst%tau_l1_bgc=tempr
+    params_inst%tau_cwd_bgc=tempr
 
-    tString='tau_l2_l3'
+    tString='cn_s1_bgc'
     call ncd_io(trim(tString),tempr, 'read', ncid, readvar=readv)
     if ( .not. readv ) call endrun(msg=trim(errCode)//trim(tString)//errMsg(sourcefile, __LINE__))
-    params_inst%tau_l2_l3_bgc=tempr
+    params_inst%cn_s1_bgc=tempr
 
-    tString='tau_s1'
+    tString='cn_s2_bgc'
     call ncd_io(trim(tString),tempr, 'read', ncid, readvar=readv)
     if ( .not. readv ) call endrun(msg=trim(errCode)//trim(tString)//errMsg(sourcefile, __LINE__))
-    params_inst%tau_s1_bgc=tempr
+    params_inst%cn_s2_bgc=tempr
 
-    tString='tau_s2'
+    tString='cn_s3_bgc'
     call ncd_io(trim(tString),tempr, 'read', ncid, readvar=readv)
     if ( .not. readv ) call endrun(msg=trim(errCode)//trim(tString)//errMsg(sourcefile, __LINE__))
-    params_inst%tau_s2_bgc=tempr
+    params_inst%cn_s3_bgc=tempr
 
-    tString='tau_s3'
+    ! TODO slevis: rf params changing
+    tString='rf_l1m1_mimics'
```

Comment:
Here we get to new parameters (with suffix _mimcs)

Reply:
again see comment above on rf (again, we only need 6).